<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>psicrom</class>
 <widget class="QDialog" name="psicrom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>BasicCommercialLTW04-Light</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PSYcrom</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>psicrom.png</normaloff>psicrom.png</iconset>
  </property>
  <property name="whatsThis">
   <string extracomment="Program to calculate  the humidity"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	color: #FFF;
	background-color:  rgb(30, 32, 37);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: 2px solid rgb(67, 149, 239);
	border-radius: 13.2px;
	color: #FFF;
	background-color:  rgb(0, 110, 255);
}

QPushButton:pressed {
	border-radius: 13.2px;
    background-color: rgb(85, 170, 255);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>Temp. seca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temp_seca">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte la temperatura seca&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 13.2px;
	color: #FFF;
	padding-left: 10px;
	background-color:  rgb(37, 36, 44);
}

QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color:  rgb(43, 45, 56);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>Punto rocio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pto_rocio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte el punto de rocio&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 13.2px;
	color: #FFF;
	padding-left: 10px;
	background-color:  rgb(34, 36, 44);
}

QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color:  rgb(43, 45, 56);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>Temperatura</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;km/h&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>Veloc. Viento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border: 2px solid rgb(67, 149, 239);
	border-radius: 13.2px;
	color: #FFF;
	background-color:  rgb(0, 110, 255);
}

QPushButton:pressed {
	border-radius: 13.2px;
    background-color: rgb(85, 170, 255);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temp_normal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte la temperatura&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 13.2px;
	color: #FFF;
	padding-left: 10px;
	background-color:  rgb(37, 36, 44);
}

QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color:  rgb(43, 45, 56);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="wind_speed">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Inserte la velocidad del viento&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 13.2px;
	color: #FFF;
	padding-left: 10px;
	background-color:  rgb(34, 36, 44);
}

QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color:  rgb(43, 45, 56);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sterm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>Humedad Relativa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BasicCommercialLTW04-Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #FFF;
	background-color:  rgb(30, 32, 37);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sensación Térmica&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>temp_seca</tabstop>
  <tabstop>pto_rocio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
